server/rest/tale.py

```diff
@@ -320,3 +322,31 @@ def stream():
             yield zip.footer()
 
         return stream
+
+    @access.public
+    @autoDescribeRoute(
+        Description('Generate the Tale manifest')
+        .modelParam('id', model='tale', plugin='wholetale', level=AccessType.READ)
+        .param(name='license',
```

Comment:
I'd be nice if that was an enum with supported *spdx*es. Maybe with default value?

Reply:
The license is now grabbed from the Tale object. Resolving.